common/database.cpp

```diff
@@ -338,6 +338,27 @@ bool Database::ReserveName(uint32 account_id, char* name) {
 	query = StringFormat("INSERT INTO `character_data` SET `account_id` = %i, `name` = '%s'", account_id, name);
 	results = QueryDatabase(query);
 	if (!results.Success() || results.ErrorMessage() != ""){ return false; }
+	
+	// Put character into the default guild if rule is being used.
+	int guild_id = RuleI(Character, DefaultGuild);
+
+	if (guild_id != 0) {
+		int char_id=-1;
+		query = StringFormat("select `id` FROM `character_data` WHERE `name` = '%s'", name);
+		results = QueryDatabase(query);
+		for (auto row = results.begin(); row != results.end(); ++row) {
+			char_id = atoi(row[0]);	
+			}
+
+		if (char_id > -1) {
+			query = StringFormat("INSERT INTO `guild_members` SET `char_id` = %i, `guild_id` = '%i'", char_id, guild_id);
```

Comment:
Please use `fmt::format` and inline it with `QueryDatabase` if it is single line

See: https://github.com/EQEmu/Server/blob/f73f72b2b2d242d4eedb904d9ce2e95bc2a5541e/zone/spawngroup.cpp#L189

Reply:
I followed the example of the code I was changing.  If I change more code, I'd asked to keep the changes as small as possible.  If I change less code, but stay with the calls being used, I'm asked to use a different call.